### ISSUE TITLE
Feature request: Expenses as Negative Numbers

### ISSUE THREAD

**Issue by zane-shus** [TomBursch/kitchenowl#441]
### What's the feature 🧐

I think its normal for people to think of income as a positive number and expenses as a negative number.  

Maybe there could be a option to have it either way.

### Extra information and references

This will enable someone to expenses against income, and see how much money they have left.

**Comment by KuLeee** [comment]
I agree, I think it would be easier for users to understand if we had income as a positive number and a negative number for expenses. This would indeed help in quickly identifying whether or not your expenses have exceeded your income for a specific month or year. This could come in the form of a toggle (on-off), even though I personally would be okay if this became the new default. Furthermore, it would be a nice addition to this if we could set specific budgets for specific months.

**Comment by zane-shus** [comment]
>  for specific months.

I would just change this to week/month/year to allow different use cases.
